specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -672,6 +745,170 @@
         }
       }
     },
+    "Marketplace": {
+      "description": "An marketplace resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/MarketplaceProperties",
+          "title": "Marketplace properties"
+        }
+      }
+    },
+    "MarketplacesListResult": {
```

Comment:
It is a good idea to explicitly express this is an object with `"type": "object"`

Reply:
@mcardosos done